### DISCUSSION TITLE
Bookmarks are clunky.

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#747] [Ideas]
### What feature would you like to see?

It would be nice if when you clicked the bookmarks star button if it opened the list of your bookmarks' toolbar that would traditionally be at the top of most browsers instead of just showing folders when you open it, as the current implementation is unusual and feels out of place with the rest of the browser.

 
 *~ Created automatically from issue: https://github.com/zen-browser/desktop/issues/716*

**Comment by AlTimofeyev** [comment]
Honestly, I agree with what @SomeDudeBrosky said in issue 716, the bookmarks bar feels a bit clunky. I just started using Zen a couple days ago and I was expecting to save bookmarks straight into the bookmark bar, but it only allows saving bookmarks into folders instead of the bar itself. From my experience, the user should be able to save bookmarks straight to the bookmark bar or any subfolders they choose to create.

Also, It'd be nice to know where the bookmarks are stored on the user's system. I'm used to chrome saving my bookmarks to my google account, but if zen doesn't do that I wanted to be able to at least copy the directory that holds all my bookmarks on my system and transfer them over to other devices that way (as a failsafe if I'm transferring to using a new machine and want to take my bookmarks with me).

**Comment by giuli635** [comment]
Well this is a thing more of Firefox than a thing of Zen, you can save it to the bookmark bar there us a folder for that, I don't know if that's what you want. Also you can sync them using Firefox Sync, and if you don't want that, you can try exploring the profile folder. I think you can find where it is under about:profiles or something similar

**Reply by giuli635** [reply]
![image](https://github.com/user-attachments/assets/66b1126b-7a50-4b9d-9578-efafb73e7c21)

Here's what I refer to, and you can activate the bookmark bar with Ctrl + Shift + B
I know, it's something weird... I don't know why there's not an option to do that under More Tools > Customize toolbar. That could be a great suggestion I think

**Reply by giuli635** [reply]
An apparently you can find you bookmarks under [profile folder]/weave/bookmarks.sqlite

**Reply by AlTimofeyev** [reply]
THANK YOU FOR THIS!!

**Comment by AlTimofeyev** [comment]
Okay. SO!! All I wanted was a quick way to access the bookmarks like I do in Chrome, from the toolbar. I tried using **ctrl+b** / **cmd+b** to access the bookmarks but it only opened the bookmark manager. So instead, I modified the bookmark keyboard shortcuts in settings to be like this:
```
Open Bookmark Manager : CTRL + SHIFT + b
Open Bookmark Toolbar : CRTL + b
```

This helps me get faster access to the bookmark toolbar at the top. I still haven't tested out how it'll look with overflow, so I'll cross that bridge when I get to it. 

**On A Side Note**  
It would still be nice if you could open the *Bookmark Toolbar* in a side pane, like with the Bookmark Manager, instead of a Top Bar. I think it could help with scrolling through tons of bookmarks without worry of overflow (cause of the scrolling). But that's only personal preference. 

Thank you again @giuli635 for pointing me in the right direction there, with the shortcuts!!

**Reply by giuli635** [reply]
You're welcome! I'm glad my comments were useful!

**Reply by AlTimofeyev** [reply]
**UPDATE**  
The `Open Bookmark Manager` and `Open Bookmark Toolbar` keyboard shortcuts are no longer settable in the settings.  
I'm currently on Version 1.0.1-a.5 (64-bit) of Zen, where the keyboard shortcuts are a little messed up at the moment.

**Comment by ghost** [comment]
### Shortcuts Folders
Good morning! I would really like to organize my home page shortcuts into folders, just like in Opera GX. I have many links that don’t fit into just 4 rows, and bookmarks don’t give me a sense of flow. I believe other people would also appreciate this feature.

![image](https://github.com/user-attachments/assets/dc45254b-db57-4232-a43c-1fac9d93f8db)
